src/design/plone/contenttypes/interfaces/servizio.py

```diff
@@ -298,17 +312,6 @@ class IServizio(model.Schema):
         ),
     )
 
-    box_aiuto = RichText(
-        title=_(u"box_aiuto", default=u"Box di aiuto"),
-        required=False,
-        description=_(
-            "box_aiuto_help",
-            default="Ulteriori informazioni sul Servizio, FAQ, eventuali"
-            " riferimenti normativi ed eventuali contatti di supporto"
```

Comment:
Come sopra per la pagina argomento. Alla fine il servizio è fatto... continuerà a funzionare e vedersi come ora, vero?

Reply:
Ci ha messo mano @giuliaghisini  perché non era proprio finito

Reply:
no, perchè il servizio era fatto ma con una sbrodolata di campi infiniti, con le voci nel menu laterale infinite e che erano molto lontane da quelle di agid. 
Anche quelli di monet ci hanno guardato e hanno suggerito diverse cose in linea con le indicazioni agid, quindi d'accordo con baio abbiamo deciso di rivederlo. Ho già sistemato anche la parte di FE, era un'attività di questa settimana per noi. Per questo c'è un branch su volto fix_view_servizio

Reply:
ottimo. importante è che siano allineati

Reply:
si i due branch (volto e plone) sono in linea per il servizio